src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/resource/UserController.kt

```diff
@@ -84,6 +88,58 @@ class UserController(
     @RequestBody @Valid
     createUserRequest: CreateUserRequest
   ): NomisUserDetails = userService.createUser(createUserRequest)
+
+  @PutMapping("/users/{userId}/enable")
+  @PreAuthorize("hasAnyRole('ROLE_MAINTAIN_OAUTH_USERS', 'ROLE_AUTH_GROUP_MANAGER')")
+  @ResponseStatus(HttpStatus.NO_CONTENT)
+  @Operation(
+    summary = "Enable a user.",
+    description = "Enable a user."
+  )
+  @ApiResponses(
+    value = [
+      ApiResponse(
+        responseCode = "204",
+        description = "OK."
+      ),
+      ApiResponse(
+        responseCode = "401",
+        description = "Unauthorized.",
+        content = [
+          Content(
+            mediaType = "application/json",
+            schema = Schema(implementation = ErrorResponse::class)
+          )
+        ]
+      ),
+      ApiResponse(
+        responseCode = "403",
+        description = "Unable to enable user, the user is not within one of your groups.",
+        content = [
+          Content(
+            mediaType = "application/json",
+            schema = Schema(implementation = ErrorResponse::class)
+          )
+        ]
+      ),
+      ApiResponse(
+        responseCode = "404",
+        description = "User not found.",
+        content = [
+          Content(
+            mediaType = "application/json",
+            schema = Schema(implementation = ErrorResponse::class)
+          )
+        ]
+      )
+    ]
+  )
+  fun enableUserByUserId(
+    @Parameter(description = "The userId of the user.", required = true) @PathVariable
+    userId: String
+  ) = userService.enableUserByUserId(
```

Comment:
This looks like a quick the same update can be done (as mentioned in external users) - i.e. pass in a UUID instead of a string

Reply:
changed to UUID